.github/workflows/build.yml

```diff
@@ -0,0 +1,213 @@
+name: build
+on:
+  push:
+    branches:
+    - master
+    - v*
+    tags:
+    - v*
+  pull_request:
+jobs:
+  build-amd64-binaries:
+    name: Build AMD64 binaries
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    # Build binaries
+    - name: Run make ci
+      run: make ci
+
+    - uses: codecov/codecov-action@v4
+      with:
+        files: ./coverage.out
+        flags: unittests
+        token: ${{ secrets.CODECOV_TOKEN }}
+
+    - name: Upload binaries
+      uses: actions/upload-artifact@v4
+      with:
+        name: binaries_amd64_artifact
+        path: ./bin/*
+
+  build-arm64-binaries:
+    name: Build ARM64 binaries
+    runs-on: ["self-hosted", "ARM64"]
```

Comment:
Use label `oracle-aarch64-2cpu-8gb` instead.

Reply:
I think we have to deploy ARM runner ourselves. The `oracle-aarch64-2cpu-8gb` can only be used in branch head. If we set it, the pull requests can't really run CI. For example, I used `oracle-aarch64-2cpu-8gb` in https://github.com/longhorn/go-spdk-helper/pull/105, but the job doesn't run.

Reply:
Update to use CNCF runners. Thanks.